testsuite/README.md

```diff
@@ -20,6 +20,7 @@ Then, you have to add some entries to your `/etc/hosts` file:
     127.0.0.1            hydra-jwt
     127.0.0.1            kafka
     127.0.0.1            mockoauth
+    127.0.0.1			 kerberos
```

Comment:
Should this have the same alignment as the lines above? I guess technically, it does not matter, but would be more readable.